CarpetX/src/io_tsv.cxx

```diff
@@ -193,7 +221,14 @@ void WriteTSVArrays(const cGH *restrict cctkGH, const std::string &filename,
     file << sep << col++ << ":"
          << "ijk"[dir];
   for (const auto &varname : varnames)
-    file << sep << col++ << ":" << varname;
+    if (cgroup.vartype == CCTK_VARIABLE_REAL ||
+        cgroup.vartype == CCTK_VARIABLE_INT)
+      file << sep << col++ << ":" << varname;
+    else if (cgroup.vartype == CCTK_VARIABLE_COMPLEX) {
+      file << sep << col++ << ":" << varname << ".real";
+      file << sep << col++ << ":" << varname << ".imag";
+    } else
+      assert(0 && "Unexpected variable type");
```

Comment:
Better call `CCTK_ERROR`

Reply:
Since this is not user (parfile or other runtime input) triggerable and the list of allowed options is determined by the flesh and not the thorn being written (CarpetX) I'd rather keep the assert (since it's a bug) rather than the `CCTK_Error`.